README.md

```diff
@@ -24,6 +24,9 @@ const Webflow = require('webflow-api');
 // Initialize the API
 const api = new Webflow({ token: 'api-token' });
 
+// Initialize the API without CORS mode
+const apiNoCors = new Webflow({ token: 'api-token', fetchOpts: { mode: undefined }});
```

Comment:
I think this is left over as well

Reply:
I don't think this is a leftover. This is how you would accomplish the removal of `cors` mode with the fetch options override approach you suggested instead of the `useCors: false` approach from the first pass.